microprofile/lra/participant/src/main/java/org/wildfly/extension/microprofile/lra/participant/MicroProfileLRAParticipantSubsystemDefinition.java

```diff
@@ -0,0 +1,105 @@
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.extension.microprofile.lra.participant;
+
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.PersistentResourceDefinition;
+import org.jboss.as.controller.ReloadRequiredRemoveStepHandler;
+import org.jboss.as.controller.ReloadRequiredWriteAttributeHandler;
+import org.jboss.as.controller.SimpleAttributeDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
+import org.jboss.as.controller.SimpleResourceDefinition;
+import org.jboss.as.controller.capability.RuntimeCapability;
+import org.jboss.as.controller.registry.AttributeAccess;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+import org.wildfly.extension.undertow.Constants;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.wildfly.extension.microprofile.lra.participant.MicroProfileLRAParticipantExtension.SUBSYSTEM_NAME;
+import static org.wildfly.extension.microprofile.lra.participant.MicroProfileLRAParticipantExtension.SUBSYSTEM_PATH;
+
+public class MicroProfileLRAParticipantSubsystemDefinition extends PersistentResourceDefinition {
+
+    private static final String LRA_PARTICIPANT_CAPABILITY_NAME = "org.wildfly.microprofile.lra.participant";
+
+    static final RuntimeCapability<Void> LRA_PARTICIPANT_CAPABILITY = RuntimeCapability.Builder
+        .of(LRA_PARTICIPANT_CAPABILITY_NAME)
+        .setServiceType(Void.class)
+        .build();
+
+    static final SimpleAttributeDefinition LRA_COORDINATOR_URL =
+        new SimpleAttributeDefinitionBuilder(CommonAttributes.LRA_COORDINATOR_URL, ModelType.STRING, true)
+            .setRequired(false)
+            .setDefaultValue(new ModelNode(CommonAttributes.DEFAULT_COORDINATOR_URL))
+            .setAllowExpression(true)
+            .setXmlName(CommonAttributes.LRA_COORDINATOR_URL)
+            .setRestartAllServices()
+            .build();
+
+    static final SimpleAttributeDefinition PROXY_SERVER =
+        new SimpleAttributeDefinitionBuilder(CommonAttributes.PROXY_SERVER, ModelType.STRING, true)
+            .setAllowExpression(true)
+            .setXmlName(CommonAttributes.PROXY_SERVER)
+            .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
+            .setDefaultValue(new ModelNode(Constants.DEFAULT_SERVER))
+            .build();
+
+    static final SimpleAttributeDefinition PROXY_HOST =
+        new SimpleAttributeDefinitionBuilder(CommonAttributes.PROXY_HOST, ModelType.STRING, true)
+            .setAllowExpression(true)
+            .setXmlName(CommonAttributes.PROXY_HOST)
+            .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
+            .setDefaultValue(new ModelNode(Constants.DEFAULT_HOST))
+            .build();
```

Comment:
This attribute appears to reference a capability from the Undertow subsystem - and thus is missing the appropriate capability reference.

Reply:
The same applies to the analogous HOST attribute in MicroProfileLRACoordinatorSubsystemDefinition.